app/extractors/meck_pomm_pdf_extractor.rb

```diff
@@ -0,0 +1,22 @@
+class MeckPommPDFExtractor
+  def initialize(paper)
+    @contents = paper.contents
+  end
+
+  # Der Minister für Bildung, Wissenschaft und Kultur hat namens der Landesregierung die Kleine Anfrage mit
+
+  ANSWERERS = /\s*(?:Der|Die)\s+(Minister.+?.*)(?:\shat\s+namens?\s+der\s+Land(?:es)?regierung\s+die)\s+(?:[kK]leine|[gG]roße)?\s*An/m
```

Comment:
"hat namen der Landregierung" looks like a lot of typos. Could you add tests with affected papers/full_references for that?